src/main/java/uk/gov/hmcts/reform/hmc/service/HearingManagementServiceImpl.java

```diff
@@ -292,20 +296,67 @@ public HearingResponse deleteHearingRequest(Long hearingId, DeleteHearingRequest
         validateVersionNumber(hearingId, deleteRequest.getVersionNumber());
         validateDeleteHearingStatus(hearingId);
         updateCancellationReasons(hearingId, deleteRequest.getCancellationReasonCode());
-        HearingEntity savedEntity = updateHearingStatus(hearingId);
+        HearingEntity savedEntity = updateHearingStatusAndVersionNumber(
+                hearingId, CANCELLATION_REQUESTED, deleteRequest);
         return getSaveHearingResponseDetails(savedEntity);
     }
 
-    private HearingEntity updateHearingStatus(Long hearingId) {
+    /**
+     * updateHearingStatusAndVersionNumber for UpdateHearingRequest.
+     * @param hearingId hearing Id
+     * @param newStatus new Status
+     * @param hearingRequest UpdateHearingRequest
+     * @return hearingEntity HearingEntity
+     */
+    private HearingEntity updateHearingStatusAndVersionNumber(Long hearingId, String newStatus,
+                                                              UpdateHearingRequest hearingRequest) {
+        HearingEntity hearingEntity = updateHearingStatusAndVersionNumber(hearingId, newStatus);
+        // Synchronise version number in request with update/not
+        hearingRequest.getRequestDetails().setVersionNumber(hearingEntity.getCaseHearingRequest().getVersionNumber());
+        return hearingEntity;
+    }
+
+    /**
+     * updateHearingStatusAndVersionNumber for DeleteHearingRequest.
+     * @param hearingId hearing Id
+     * @param newStatus new Status
+     * @param hearingRequest DeleteHearingRequest
+     * @return hearingEntity HearingEntity
+     */
+    private HearingEntity updateHearingStatusAndVersionNumber(Long hearingId, String newStatus,
```

Comment:
This method is not required as updating the request is not needed.

Reply:
Agreed.